koku/masu/database/report_db_accessor_base.py

```diff
@@ -115,11 +113,13 @@ def create_new_temp_table(self, table_name, columns):
         column_types = column_types.strip().rstrip(',')
         column_sql = '({})'.format(column_types)
         table_sql = base_sql + column_sql
-        self._cursor.execute(table_sql)
+        with connection.cursor() as cursor:
+            cursor.db.set_schema(self.schema)
+            cursor.execute(table_sql)
 
         return temp_table_name
 
-    # pylint: disable=too-many-arguments
+    # pylint: disable=too-many-arguments,too-many-locals
```

Comment:
Instead of disabling the `too-many-locals` check, let's see if we can reduce the number of local variables in this function.

I can already spot a couple of places where a local variable is being assigned when it's not needed.